apns/transport.go

```diff
@@ -0,0 +1,19 @@
+package apns
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// RoundTripperJWTDecorator an implementation of http.RoundTripper interface
+// with ability to specify authorization JWT token for each request.
+type RoundTripperJWTDecorator struct {
+	Parent http.RoundTripper
+	Token  *Token
+}
+
+func (d *RoundTripperJWTDecorator) RoundTrip(r *http.Request) (*http.Response, error) {
+	r.Header.Set("Authorization", fmt.Sprintf("bearer %s", d.Token.Bearer()))
```

Comment:
ifaik token should be refreshed every <timeout>.

Reply:
it should be refreshed every hour, the library guarantees refresh every 50 mins